examples/active_support_helper.rb

```diff
@@ -0,0 +1,30 @@
+module ActiveRecord
+  class Base
+  end
+end
+
+module ActiveSupport
+  class TestCase < Minitest::Test
+    def setup(*args, &block)
+    end
+
+    # Remove describe method if present
+    class << self
```

Comment:
use of a multi-line postfix if clause here seems harder to read than wrapping the thing in an if. Also is the `class << self` strictly necessary if it's only used for one entry?

Could this say instead:

``` ruby
    if self.respond_to?(:describe) && self.method(:describe).owner == ActiveSupport::TestCase
      self.class.remove_method :describe
    end
```

(just shooting from the hip, didn't test it)